lua/earmuffs/client/cl_settings.lua

```diff
@@ -0,0 +1,77 @@
+AddCSLuaFile()
+
+CFCEarmuffs = CFCEarmuffs or {}
+CFCEarmuffs.Settings = CFCEarmuffs.Settings or {}
+local logger = CFCEarmuffs.logger
+
+local SETTINGS_DEFAULTS = {
+    CombatVolumeMult = 0.2,
+    CombatSoundLevelMult = 0.2
+}
+
+local SETTINGS_NAMES = {
+    CombatVolumeMult = "CFC_Earmuffs_CombatVolumeMult",
+    CombatSoundLevelMult = "CFC_Earmuffs_CombatSoundLevelMult"
+}
```

Comment:
This and `SETTINGS_DEFAULTS` would probably be nicer as a bigger table structure in another file, for example:
```lua
CFCEarmuffs.SETTINGS = {
    {
        name = "CombatVolumeMult",
        type = "float",
        default = 0.2
    }
}
```
This allows us to add more settings with different types more easily.
I've already done this kind of settings saving [here](https://github.com/CFC-Servers/cfc_notifications/blob/new-structure/lua/cfc_notifications/client/cl_settings.lua#L33)
These can then be automated to use convars